dev/tidb-in-kubernetes/reference/tools/tools-in-kubernetes.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Tools in Kubernetes
+summary: Learn about operation tools for TiDB in Kubernetes.
+category: reference
+---
+
+# Tools in Kubernetes
+
+Operations on TiDB in Kubernetes require some open source tools. In the meantime, there are some special requirements for operations using TiDB tools in the Kubernetes environment. This documents introduces in details about related operation tools for TiDB in Kubernetes.
+
+## Use PD Control in Kubernetes
+
+[PD Control](reference/tools/pd-control.md) is the command line tool for PD (Placement Driver). To use PD Control to operate on TiDB clusters in Kubernetes, firstly you need to establish the connection from local to the PD service using `kubectl port-forward`:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl port-forward -n <namespace> svc/<cluster-name>-pd 2379:2379 &>/tmp/portforward-pd.log
+```
+
+After the above command is executed, you can access the PD service via `127.0.0.1:2379`, and then use the default parameters of `pd-ctl` to operate. For example:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+pd-ctl -d config show
+```
+
+Assume that your local port `2379` has been occupied and you want to switch to another port. Now you need to explicitly assign a PD port for `pd-ctl`:
+
+```shell
+pd-ctl -u 127.0.0.1:<port> -d config show
+```
+
+## Use TiKV Control in Kubernetes
+
+[TiKV Control](reference/tools/tikv-control.md) is the command line tool for TiKV. When using TiKV Control for TiDB clusters in Kubernetes, be aware that different operation mode may involve different steps.
+
+* **Remote Mode**: In this mode, `tikv-ctl` accesses the TiKV service or the PD service through network, so firstly you need to establish the connection from local to the PD service and the target TiKV node using `kubectl port-forward`:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl port-forward -n <namespace> svc/<cluster-name>-pd 2379:2379 &>/tmp/portforward-pd.log
+    ```
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl port-forward -n <namespace> <tikv-pod-name> 20160:20160 &>/tmp/portforward-tikv.log
+    ```
+
+    After the connection is established, you can access the PD service and the TiKV node via the corresponding port in local:
```

Comment:
```suggestion
    After the connection is established, access the PD service and the TiKV node via the corresponding port in local:
```

Reply:
This is a description to indicate a result of the previous operations - that you can do something. So I think we should keep "you can" here